www/docs/src/content/docs/how-it-works/index.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "How does it work!?"
+description: "How does Astro StudioCMS Work?  Well here is all the answers!"
+---
+
+# Introduction
+
+StudioCMS is a CMS built for Astro. It is a powerful tool that allows you to create, edit, and delete content for your Astro project in an SSR deployment while connected to [Astro Studio](https://studio.astro.build). It is designed to be easy to use and flexible enough to handle any type of content. And it is easy to extend! You can also create your own plugins to add new features to StudioCMS.
+
+## How does it work?
+
+AstroStudioCMS is devided into multiple parts:
```

Comment:
```suggestion
StudioCMS is devided into multiple parts:
```

Don't want to keep flip-flopping between `AstroStudioCMS` and `StudioCMS`. Don't necessarily mind which we use as long as we're consistent. Though `Astro StudioCMS` is the full name, I feel like `StudioCMS` is fine throughout the docs